db/migrate/20191108153914_add_hidden_by_commentable_user_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddHiddenByCommentableUserToComments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :comments, :hidden_by_commentable_user, :boolean, default: false
```

Comment:
Probably doesn't need an index right?

Reply:
This is correct. If we had a `where(hidden_by_commentable_user: true)` call somewhere we might want one.